lib/streamlit/runtime/pages_manager.py

```diff
@@ -209,6 +198,8 @@ def __init__(self, main_script_path, script_cache=None, **kwargs):
         self._current_page_hash: PageHash = self._main_script_hash
         self.pages_strategy = PagesManager.DefaultStrategy(self, **kwargs)
         self._script_cache: ScriptCache | None = script_cache
+        self._intent_page_script_hash: PageHash | None = None
+        self._intent_page_name: PageName | None = None
```

Comment:
nit: Should these names be `intended*` rather than `intent*` throughout? It feels like that form of the word is likely slightly more correct here

Reply:
I think both convey the intent (pun intended....x2 😆), but I am happy to put the more correct form here.